packages/dds/tree/src/core/tree/delta.ts

```diff
@@ -176,6 +176,7 @@ export interface Modify<TTree = ProtoNode> {
 	readonly type: typeof MarkType.Modify;
 	readonly setValue?: Value;
 	readonly fields?: FieldMarks<TTree>;
+	readonly valueConstraint?: Value;
```

Comment:
We shouldn't need to describe constraints in deltas.

Reply:
You're right, removed